### ISSUE TITLE
Replace technologies list with icons

### ISSUE THREAD

**Issue by EileenMcCall** [EileenMcCall/eileen-mccall-portfolio-next#4]
List of technologies on homepage should be a series of logos, with sr-only text descriptions for accessibility
